convert2rhel/pkghandler.py

```diff
@@ -671,6 +672,7 @@ def get_kernel_availability():
     """Return a tuple - a list of installed kernel versions and a list of
     available kernel versions.
     """
+# is tuple spelled correctly? This is a math term.
```

Comment:
Yes, it's a python data type (a read-only list): https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences  Thanks for verifying!